manuscript/18-EngFound.txt

```diff
@@ -121,8 +122,798 @@
+προσπάθεια και συνεπώς αποτελούν στοιχείο των περιορισμών που επιβάλλονται σε κάθε προσπάθεια σχεδιασμού. Ο 
+μηχανικός πρέπει να επανεξετάσει όλα τα ισχύοντα πρότυπα που σχετίζονται με μια δεδομένη προσπάθεια και να καθορίσει 
+ποια πρέπει να τηρηθούν. Τα σχέδιά τους πρέπει στη συνέχεια να ενσωματώνουν όλους τους περιορισμούς που επιβάλλονται από το ισχύον πρότυπο.
+
+## **Ανάλυση αιτιών**
+Η ανάλυση βαθύτερων αιτιών (RCA) είναι μια κατηγορία 
+μεθόδων επίλυσης προβλημάτων με στόχο 
+τον εντοπισμό των υποκείμενων αιτιών των ανεπιθύμητων 
+αποτελεσμάτων. Οι μέθοδοι RCA προσδιορίζουν τους λόγους και 
+πώς συνέβη ένα ανεπιθύμητο αποτέλεσμα, 
+επιτρέποντας στους οργανισμούς να λάβουν αποτελεσματικές
+ενέργειες για την αποτροπή της επανάληψης. Αντί για
+την απλή αντιμετώπιση των άμεσα προφανών 
+συμπτωμάτων, τα προβλήματα μπορούν να επιλυθούν 
+εξαλείφοντας τα βαθύτερα αίτια. Η RCA μπορεί να παίξει 
+διάφορους σημαντικούς ρόλους σε έργα λογισμικού, 
+μεταξύ άλλων:
+
+* τον εντοπισμό του πραγματικού προβλήματος που πρέπει να επιλυθεί από μία μηχανική προσπάθεια,
+
+* την αποκάλυψη των υποκείμενων παραγόντων κινδύνου κατά την εκτέλεση αξιολογήσεων κινδύνου έργου,
+
+* την αποκάλυψη ευκαιριών και δράσεων για τηνβελτίωση της διαδικασίας λογισμικού,
+
+* την ανακάλυψη πηγών επαναλαμβανόμενων ελαττωμάτων (δηλ. αιτιώδης ανάλυση ελαττωμάτων).
+
+### *Τεχνικές ανάλυσης αιτιών*
+
+Υπάρχουν διάφορες τεχνικές RCA, μεταξύ των οποίων:
+
+* Η ανάλυση αλλαγής συγκρίνει καταστάσεις όπου 
+συνέβη ένα ανεπιθύμητο αποτέλεσμα με 
+παρόμοιες καταστάσεις όπου δεν συνέβη. Η κύρια
+αιτία είναι πιθανό να βρίσκεται στην περιοχή της διαφοράς,
+
+* Το 5-Γιατί ξεκινάει 
+με ένα ανεπιθύμητο αποτέλεσμα και χρησιμοποιεί 
+επαναλαμβανόμενους κύκλους ερωτήσεων-απαντήσεων "Γιατί;" για να 
+να απομονώσει τη βασική αιτία, 
+
+* Διαγράμματα αιτίας και αποτελέσματος, μερικές φορές 
+που ονομάζονται διαγράμματα Ishikawa [15] ή Fishbone 
+διαγράμματα, αναλύουν, σε διαδοχικά επίπεδα 
+τις αιτίες που δυνητικά συμβάλλουν σε ένα 
+ανεπιθύμητο αποτέλεσμα. Οι αιτίες  συχνά 
+ομαδοποιούνται σε μεγάλες κατηγορίες, όπως οι άνθρωποι, 
+η διαδικασία, τα εργαλεία, τα υλικά, οι μετρήσεις και 
+το περιβάλλον. Το διάγραμμα έχει τη μορφή 
+ενός δέντρου πιθανών αιτιών που μπορούν να οδηγήσουν σε 
+αυτό το ανεπιθύμητο αποτέλεσμα,
+
+* Η Ανάλυση Δέντρου Σφαλμάτων (ΑΔΣ) είναι μια πιο επίσημη 
+προσέγγιση στη διαγραμματική απεικόνιση αιτίου-αποτελέσματος 
+η οποία είναι πιο συγκεκριμένη σχετικά με τις  "και-ή"
+σχέσεις μεταξύ αιτιών και αποτελεσμάτων. Σε 
+ορισμένες περιπτώσεις, οποιαδήποτε από τις πολλαπλές αιτίες μπορεί 
+να οδηγήσει στο αποτέλεσμα (μια σχέση "ή"), σε άλλες  
+περιπτώσεις ένας συνδυασμός πολλαπλών αιτιών είναι 
+για να οδηγήσει το αποτέλεσμα (μια σχέση "και"). Τα διαγράμματα αιτίας και αποτελέσματος 
+δεν διακρίνουν μεταξύ των σχέσεων "και-ή", 
+Η ανάλυση δέντρων σφαλμάτων το κάνει αυτό,
+
+* Η αξιολόγηση της ανθρώπινης απόδοσης υποστηρίζει ότι 
```

Comment:
λείπει τοFailure Modes and Effects Analysis
(FMEA) , • Cause Map [16], • Current Reality Tree. Δεν τα βρίσκω;